### ISSUE TITLE
Rakefile broken for Debian Ruby 1.9.2

### ISSUE THREAD

**Issue by jorrizza** [shoes/shoes-deprecated#70]
I can't build Shoes properly on Debian using Ruby 1.9.2. Gems are included and `.` is not part of `$:` anymore. After applying this patch it does work using `rake1.9.1`:

```
diff --git a/Rakefile b/Rakefile
index b4513c7..b092038 100644
--- a/Rakefile
+++ b/Rakefile
@@ -1,7 +1,8 @@
-require 'rubygems'
+$:.push File.dirname(__FILE__)
+
 require 'rake'
 require 'rake/clean'
-require_relative 'platform/skel'
+require 'platform/skel'
 require 'fileutils'
 require 'find'
 require 'yaml'
```

I'm not entirely sure whether this is Debian-specific. But since you mention Debian on the wiki, maybe this patch belongs there.

**Comment by steveklabnik** [comment]
It's not debian specific, it's because Shoes is currently 1.9.1 only.

1.9.2 work is underway on a branch here: https://github.com/shoes/shoes/tree/feature/1.9.2

I have a personal todo to make the '1.9.1 only' stuff more clear in the documentation... I'm sorry this was confusing. The whole wiki needs cleanup...

**Comment by jorrizza** [comment]
Whoops, sorry I've missed that. The current master branch works flawlessly in 1.9.2p0 as far as I've tested it, except for the Rakefile anyway.

**Comment by steveklabnik** [comment]
No worries at all. Our documentation is pretty poor at the moment.

With those rakefile fixes, it should build with 1.9.2, yes. the only reason it hasn't been committed to master yet is that we also want shoes to actually use 1.9.2 for the next release as well, and that will require some more research, work, and testing....

But maybe that patch should be moved onto master first, anyway. You're not the first person to report this. Hm.

**Comment by steveklabnik** [comment]
In response to this, I've merged the 1.9.2 building fix into master. It'll help people like you out in the meantime, until we get full 1.9.2 support within Shoes.

Thanks again for pointing this out.
